lib/backbone-provider.js

```diff
@@ -9,7 +11,7 @@ class BackboneProvider extends Component {
   }
 
   render() {
-    return Children.only(this.props.children);
+    return createElement(Provider, { value: this.props.models }, Children.only(this.props.children));
```

Comment:
It felt weird needing to use `createElement` instead of JSX, but we don't appear to have the eslint React plugin set up, so it was complaining about `Provider` not being used otherwise.

Reply:
I was assuming `eslint-plugin-react`, specifically [react/jsx-uses-react](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-react.md) and [react/jsx-uses-vars](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-vars.md), but maybe my editor was just reading the linter config wrong? Tracing it manually, it looks like we should be eventually inheriting from airbnb, so those rules should already be included...